tests/data/test3026

```diff
@@ -15,11 +15,8 @@ slow
 #
 # Client-side
 <client>
-# require the threaded resolver only because it means pthreads might
-# be used for it
 <features>
-threaded-resolver
```

Comment:
It still needs the threaded-resolver feature present as otherwise we don't know it has thread support present.

Reply:
That may be true for Linux, but I don't think this is correct for Windows builds, right?

Reply:
Right, it's primarily for non-Windows reasons but then again there aren't many windows builds done without `threaded-resolver`.

Reply:
I see. The test will fail for non-Windows builds of libcurl that don't link pthreads.

Reply:
Okay, then I will keep `threaded-resolver`.